machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -29,7 +29,7 @@ In {product-title} {product-version}, the Control Plane Machine Set Operator is
 
 The status of the Operator after installation depends on your cloud provider and the version of {product-title} that you installed on your cluster.
 
-.Control Plane Machine Set Operator implementation matrix
+.Control plane machine set implementation for {product-title} {product-version}
```

Comment:
Revised table to show only 4.13 status, and added `{product-title} {product-version}` to hopefully make that clear. @odvarkadaniel what do you think, better now?

Reply:
LGTM!